libmscore/cmd.cpp

```diff
@@ -1779,12 +1779,21 @@ bool Score::processMidiInput()
       if (MScore::debugMode)
           qDebug("processMidiInput");
 
+      NoteEntryMethod entryMethod = _is.noteEntryMethod();
       bool cmdActive = false;
       while (!midiInputQueue()->empty()) {
             MidiInputEvent ev = midiInputQueue()->dequeue();
+            for (std::list<MidiInputEvent>::iterator itr = activeMidiPitches()->begin(); itr != activeMidiPitches()->end();) {
```

Comment:
You can probably use `auto`. It will be a lot more readable.
or (auto itr = activeMidiPitches()->begin(); itr != activeMidiPitches()->end();) {